src/TBuffer.cpp

```diff
@@ -51,529 +51,530 @@
 // clang-format off
```

Comment:
I also used uncrustify on it to add single-line braces for you - so it would be that tool that did the changes. The formatting looks OK though.